### PR TITLE
feat: add kvstore into monorepo

### PR DESCRIPTION
**Pull request by spiritupbro**
closes https://github.com/humanprotocol/human-protocol/issues/64

### CODE REVIEW

**Review by m00n620 — APPROVED** [review]

**Comment by spiritupbro** [comment]
@m00n620 thanks for the review, will do that for sure

**Review by portuu3 — CHANGES_REQUESTED** [review]
I get missing token after generating the key. I cannot go forward after this 
image

Also need tests

**Comment by spiritupbro** [comment]
> I get missing token after generating the key. I cannot go forward after this

hello @portuu3 i forgot to mention you need to add additional environment variable in `.env`

```
REACT_APP_NFT_STORAGE_API=
REACT_APP_CONTRACT=0x05877184aB2ddAb0F9D1fa6c573392fCe7A7a74a
REACT_APP_ALCHEMY_API_KEY=
```

i will try to upload it to my own cloud so you can try actually after clicking store eth kvstore it will store then nothing happens because i still waiting the design from @ansaqib for encryption and decryption, basically it all working just like my eth-kvstore web gui just different design

> Also needs tests

ok will add test for both component and contract

**Comment by spiritupbro** [comment]
> @spiritupbro perfect thanks

@portuu3 https://escrow-dashboard-spiritbro.vercel.app/kvstore here you go, also i want to ask in the monorepo for apps/escrow-dashboard there is no contract deployment so i assume the test you meant here is only component test isn't it?

**Comment by portuu3** [comment]
> > @spiritupbro perfect thanks
> 
> @portuu3 https://escrow-dashboard-spiritbro.vercel.app/kvstore here you go, also i want to ask in the monorepo for apps/escrow-dashboard there is no contract deployment so i assume the test you meant here is only component test isn't it?

Yes, the contract is deployed in `core`, don't worry about contract test

**Comment by spiritupbro** [comment]
Hello @portuu3 could you help me a little bit? Im stuck, so basically the app is finish i already convert the design to frontend but i encountered issue with testing, wagmi use es module, and jest doesnt want to transform the es module to commonjs, ive tried added transform to jest.config.js but it doesnt work do you know how to resolve this issue?

**Comment by spiritupbro** [comment]
@portuu3 @m00n620 please review test and ui done

File: packages/apps/escrow-dashboard/.env.example [hunk 1]
```diff
@@ -1 +1,5 @@
 PORT=3002
+REACT_APP_NFT_STORAGE_API=
+REACT_APP_CONTRACT=0x05877184aB2ddAb0F9D1fa6c573392fCe7A7a74a
+REACT_APP_ALCHEMY_API_KEY=
+DISABLE_ESLINT_PLUGIN=
```
Comment (portuu3) [inline packages/apps/escrow-dashboard/.env.example:5]: remove this

**Review by portuu3 — CHANGES_REQUESTED** [review]
@spiritupbro  Pull changes from main branch and solve conflicts.
About eth-kvstore contract address you should add kvstore address to the interface `IEscrowNetwork` [here](https://github.com/humanprotocol/human-protocol/blob/main/packages/apps/escrow-dashboard/src/constants/index.ts), and then add to the list of networks their kvstore address that you can find [here](https://github.com/humanprotocol/human-protocol/blob/main/CONTRACTS_LIST.md).
Eth kvstore should use each address according to he network selected in the wallet.
Execute `yarn lint` and fix lint warnings
Check Alchemy comment

File: packages/apps/escrow-dashboard/src/constants/index.ts [hunk 2]
```diff
@@ -12,7 +25,31 @@ export enum ChainId {
...
   AVALANCHE = 43114,
 }
-
+const rinkeby = {
+  id: ChainId.RINKEBY,
+  name: 'Ethereum Rinkeby',
+  network: 'rinkeby',
+  nativeCurrency: {
+    decimals: 18,
+    name: 'Avalanche',
```
Comment (portuu3) [inline packages/apps/escrow-dashboard/src/constants/index.ts:34]: you are using Avalanche for rinkeby, anyway Rinkeby network shoudn't be available in the kvstore, this network is deprecated

**Review by portuu3 — CHANGES_REQUESTED** [review]

File: packages/apps/escrow-dashboard/src/constants/index.ts (hunk 2 above)
Comment (spiritupbro) [reply packages/apps/escrow-dashboard/src/constants/index.ts:34]: Ah sorry let me change that

**Review by portuu3 — APPROVED** [review]

**Review by m00n620 — APPROVED** [review]

**Pull request merged**